README.md

```diff
@@ -38,7 +38,7 @@ Firmware binaries can be downloaded from http://ota.tasmota.com/tasmota/release/
 
 See [CHANGELOG.md](CHANGELOG.md) for detailed change information.
 
-Unless your Tasmota powered device exhibits a problem or you need to make use of a feature that is not available in the Tasmota version currently installed on your device, leave your device alone - it works so don't make unnecessary changes! If the release version (i.e., the master branch) exhibits unexpected behaviour for your device and configuration, you should upgrade to the latest development version instead to see if your problem is resolved as some bugs in previous releases or development builds may already have been resolved.
+Unless your Tasmota powered device exhibits a problem or lacks a feature that you need, leave your device alone - it works so don’t make unnecessary changes!If the release version (i.e., the master branch) exhibits unexpected behaviour for your device and configuration, you should upgrade to the latest development version instead to see if your problem is resolved as some bugs in previous releases or development builds may already have been resolved.
```

Comment:
I would like the keep the space after the exclamation mark and the next uppercase character of "If". Normal in europe to have spaces here ;-) and increases readability too.

Reply:
ooh interesting ok!